app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java

```diff
@@ -55,4 +59,22 @@ public boolean isFullscreenForTabOrPending() {
     public int getDisplayMode() {
         return !mIsFullscreen ? DisplayMode.BROWSER : DisplayMode.FULLSCREEN;
     }
+
+    @Override
+    public void navigationStateChanged(int flags) {
+        if (((flags) & InvalidateTypes.TITLE) != 0) {
+            @Nullable WSession.ContentDelegate delegate = mSession.getContentDelegate();
```

Comment:
Hmm we don't normally use the `@Nullable` for variables just for method definitions. Is it somehow required? If not please remove it

Reply:
Done